packages/cursorless-engine/src/scripts/generateTestSubsetFile/generateTestSubsetFile.ts

```diff
@@ -0,0 +1,43 @@
+import { getCursorlessRepoRoot } from "@cursorless/common";
+import * as child from "child_process";
+import * as fs from "fs";
+import * as path from "path";
+
+const template = `# This file contains the grep strings to pass to Mocha when running a subset of tests.
+# These grep strings will be used with the "Run test subset" launch configuration.
+# See https://mochajs.org/#-grep-regexp-g-regexp for supported syntax.
+#
+# One regular expression per line.
+# Tests matching any of the regular expressions will be run.
+
+snippets
+languages/go/
+`;
+
+function run() {
+  // Two possible modes:
+  //   * user explicitly asked us to open; create as needed, always open
+  //   * run as a side effect of "test subset"; create as needed, and open and fail if so
+  const testSubsetGrepPath = path.join(
```

Comment:
Should I dedup this by exporting a function in testUtil?

Reply:
I'll do that in a follow-up; we should migrate all this stuff into one place; no reason to have it spread over three packages when it's all consumed in one package

Reply:
thanks :)